src/common/store.ts

```diff
@@ -2,9 +2,11 @@ import { configureStore } from '@reduxjs/toolkit';
 import { combineReducers } from 'redux';
 
 import search from '../reducers/SearchSlice';
+import compareTable from '../reducers/SelectedRevisions';
 
 const reducer = combineReducers({
   search,
+  compareTable,
```

Comment:
`compareRevisions` and `compareTable` don't really reflect what this table is, and I think it would overlap with names used for the compare view. Please use `selectedRevisions` instead